src/components/ATATDialog.spec.ts

```diff
@@ -0,0 +1,61 @@
+import Vue from "vue";
+import Vuetify from "vuetify";
+import {createLocalVue, mount, Wrapper} from "@vue/test-utils";
+import ATATDialog from "@/components/ATATDialog.vue";
+import {DefaultProps} from "vue/types/options";
+
+
+Vue.use(Vuetify);
+
+describe("Testing ATATDialog Component", () => {
+  const localVue = createLocalVue();
+  let vuetify: Vuetify;
+  let wrapper: Wrapper<DefaultProps & Vue, Element>;
+
+  beforeEach(() => {
+    vuetify = new Vuetify();
+    jest.useFakeTimers();
+    wrapper = mount(ATATDialog, {
+      localVue,
+      vuetify,
+      propsData: {
+        title: '',
+      }
+    });
+  });
+  it("renders successfully", async () => {
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  describe("PROPS", () => {
+    it("optional", async()=>{
+      await wrapper.setProps({title: `Bacon ipsum dolor amet ball tip minim shoulder
+         buffalo t-bone eu.Strip steak nostrud pancetta ullamco
```

Comment:
indent is one space too much on 33 and 34... actually 34 can just go up onto 33, no need for 2 lines. The text makes me hungry!